Files/powerGate/Modules/Communication.psm1

```diff
@@ -2,12 +2,31 @@
 Import-Module powerGate
 
 #TODO: configure the powerGate server url and port
-$powerGateServerName = $ENV:Computername
+$powerGateServerName = getRelatedPGServerName
 $powerGateServerPort = "8080"
+$vaultDefinitions = @{
+	"TEST" = "TestVault";
+	"PROD" = "ProductiveVault"
+}
+$PGServerDefinitions = @{
+	"TEST" = "TestERP";
```

Comment:
Since this is a configuration mapping here.
**ToDo**: Please add comments that not the scripter know what they need to configure here.

Also **ToDO** the default configured value should be `$ENV:Computername` Since this is a template and it should work without configuring we should not change the fact that the default value is `$ENV:Computername`

Reply:
See my last comment on this line: The default value should be configured here.